chart/templates/scheduler/scheduler-deployment.yaml

```diff
@@ -119,6 +119,7 @@ spec:
         - name: {{ template "registry_secret" . }}
       {{- end }}
       initContainers:
+        {{- if .Values.scheduler.waitForMigrations.enabled }}
```

Comment:
Why would we not want to check that the db is in the right state before starting? I get being able to disable the job itself, but I'm torn on this.

Reply:
For our use case, we run the db migrations within the `entrypoint.sh` script. It is less complicated than doing it in a k8s `Job` and because we're running it there, we don't need the `initContainer`